versioned_docs/version-2022.3/03_server/05_access-control/04_sso_authentication.md

```diff
@@ -35,11 +35,10 @@ The `sso` function allows you to configure and enable SSO options. It has the fo
   * In the case of `NewUserMode.CREATE_ENABLED`, when a user logs in for the first time with SSO, if they do not already have a user account, an active account is created for them.
   * In the case of `NewUserMode.CREATE_DISABLED`, when a user logs in for the first time with SSO, if they do not already have a user account, a disabled account is created for them. This will be need to be activated before it can be used.
 
-### passwordRetry
```

Comment:
This felt like the wrong place because you can't actually configure `passwordRetry` in the `sso` block, so I changed it to a note - this property is deprecated since the last version too, so I've noted it as such on the latest two versions and changed the link to point to the new `retry` property in `genesisPassword`